add Strapi (#37)
thisdot/tech-dev-tooling: README.md

- [Hasura](https://hasura.io/)
  Hasura gives you instant GraphQL & REST APIs on new & existing data sources. Connect Hasura to your data & get APIs in under a minute.

- [Strapi](https://strapi.io/)

>   Strapi is a open-source CMS (Content Management Systema) with a beauty and fully customizable dashboard. Build the API faster (REST or GraphQL) and manage your content easily.

Reviewer: ```suggestion
  Strapi is an open-source CMS (Content Management System) with a beautiful and fully customizable dashboard. Build your API's faster (REST or GraphQL) and manage your content easily.
```
